CHANGELOG.md

```diff
@@ -0,0 +1,86 @@
+## [0.0.1-alpha.0](https://github.com/Wynntils/Artemis/compare/v0.0.0...v0.0.1-alpha.0) (2022-11-02)
```

Comment:
Do we need a changelog file? To me it seems like just a burden to have it updated all the time. *Especially* if we now have commits with the markers that mean we can generate it automatically.

The one thing I can think of as an exception is if we want to include free text overviews, like we have done in Legacy. "Hi everyone, In this release we have focused on the ...." etc.

Reply:
It will auto update from the build system with the last `5` version changelogs by default. Take a look at https://github.com/Wynntils/Wynntils/blob/development/CHANGELOG.md